### DISCUSSION TITLE
RNE Slider with marker and Input field

### DISCUSSION THREAD

**Discussion by patelvimal** [react-native-elements/react-native-elements#3451] [Ideas]
While developing my react native application , I come across a situation where I wanted to add marker and input field . I was already using your RNE library. Hence I created a custom component on top of your slider component.

[Input Slider with Marker](https://drive.google.com/file/d/1vZGG6VnWTjOi2xp2xdRz3KjcLXfQumWY/view?usp=sharing)

I would love if you guys can incorporate this component in your library so that it can benefit others as well.
Let me know your thoughts.

**Comment by flyingcircle** [comment]
@patelvimal I think this would be a cool addition. If you have a PR available to review, that would be great.

**Comment by patelvimal** [comment]
I don't have a PR as of now, However give me few days, will do that.

**Comment by hetanthakkar** [comment]
Work in progress
